test/unit/concerns/symbolize_test.rb

```diff
@@ -50,7 +50,8 @@ def test_period_previous_change
   end
 
   test '#changes is symbolized but not #previous_changes' do
-    Timecop.freeze(Time.zone.now.round) do
+    travel_back
```

Comment:
I am not sure why these tests use time traveling in the first place, the module does not seem to be depending on any timestamps. I'd suggest remove these travel methods,and also `setup` and `teardown`.

Reply:
You're right, this is not needed neither